src/skmultiflow/utils/data_structures.py

```diff
@@ -1014,3 +1015,85 @@ def get_info(self):
                ' - n_samples: ' + str(self.n_samples) + \
                ' - max_size: ' + str(self.max_size) + \
                ' - dtype: ' + str(self.dtype)
+
+class TimeManager(object):
```

Comment:
I'd add more docstring to this class.

Reply:
Fixed in commit [47646ebb1bd25782655df2fc8b38930df6971f5d](https://github.com/scikit-multiflow/scikit-multiflow/commit/47646ebb1bd25782655df2fc8b38930df6971f5d).